specification/2.0/appendix/performance-metrics/README.md

```diff
@@ -0,0 +1,107 @@
+# Runtime Performance Metrics 
+
+The goal of these metrics is to assess the runtime performance needs of glTF assets.  
+By runtime we mean the cpu/gpu, bandwidth and memory resources needed to render the model in realtime.  
+These performance metrics are divided in two major categories - `complexity` and `memory`
+
+## Performance Metric Categories
+
+### Complexity  
+
+The `complexity`category provides metrics to calculate cpu/gpu and memory bandwidth target requirements - larger numbers will mean a more powerful target device is needed.  
+These metrics do not provide a mapping to target device performance, this mapping must be done by users of the metrics.  
+
+### Screen space coverage
+
+The metrics do not include screen space coverage, it is advised that clients shall know how large the render area will be before requesting a model (from the backend)  
+It is up to the backend to take screen coverage into account when providing a suitable model to the client, for instance by taking width + height as parameters.  
+
+### Memory  
+
+The `memory` category provides metrics to calculate the runtime memory (size) needs of the model.  
+Using these metrics an estimate of target memory can be calculated - however keep in mind that the exact memory requirements will vary with viewer implementations.  
```

Comment:
A small nit — the metrics below are not exclusively about memory use. Primitive count mainly affects draw call count (and thus CPU time). A distinction could be drawn between CPU and GPU memory, but I don't think it's necessary. Some indication of GPU time might also be helpful... that's hard to measure, but i'll attempt some suggestions below.

Reply:
I meant that the `memory` category of metrics are for memory. 
The `primitiveCount` is in the complexity category - is this not explained properly @donmccurdy